x-pack/legacy/plugins/maps/public/connected_components/map/mb/utils.js

```diff
@@ -74,36 +33,42 @@ export function removeOrphanedSourcesAndLayers(mbMap, layerList) {
 }
 
 export function syncLayerOrder(mbMap, layerList) {
```

Comment:
no changes to the implementation, just flips the guard statement, to handle the shortcut case first.